src/main/java/fi/okm/jod/yksilo/repository/OsaaminenRepository.java

```diff
@@ -56,6 +51,9 @@ default SequencedMap<URI, OsaaminenDto> loadAll() {
   @Query("SELECT v.versio FROM OsaaminenVersio v WHERE v.id = 1")
   long currentVersion();
 
+  @Query("SELECT o FROM Osaaminen o JOIN FETCH o.kiinnostuneet y WHERE y.id IN :yksilot")
+  List<Osaaminen> fetchOsaamisKiinnostukset(List<UUID> yksilot);
+
```

Comment:
Seems unused. Note that without the bi-directional association, you could write this query ~
`SELECT o FROM Yksilo y JOIN y.osaamisKiinnostukset o WHERE y.id IN :yksilot`